tests/tests_pytorch/strategies/test_colossalai.py

```diff
@@ -120,7 +120,7 @@ def test_gradient_accumulation_error(tmpdir):
         trainer.fit(model)
 
 
-@RunIf(min_cuda_gpus=1, colossalai=True)
+@RunIf(min_cuda_gpus=1, standalone=True, colossalai=True)
```

Comment:
Why did you make this standalone? This just checks, whether appropriate error is raised, which is happening regardless of standalone.